package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "tracking-wallet",
-  "version": "5.3.6",
+  "version": "5.3.7",
```

```suggestion
  "version": "5.4.0",
```
This PR is adding a new feature, not fixing a bug ([SEMVER](https://semver.org/) is very important in this library)